assets/js/theme/global/modal.js

```diff
@@ -196,12 +210,65 @@ export class Modal {
         this.$content.html('');
     }
 
+    setupFocusableElements(modalType) {
+        this.$lastPageFocusedElement = $(document.activeElement);
+
+        const $collection = focusableElements[modalType]();
+        $collection.each((index, element) => $(element).removeClass(focusedClass)); 
+        const $elementToFocus = $($collection.get(0));
+        $elementToFocus.focus().addClass(focusedClass);
+
+        $('#modal').on('keydown', this.onTabbing.bind(null, modalType));
+    }
+
+    onTabbing(modalType, event) {
```

Comment:
I suggest to use currying for onTabbing func modalType => event => {}

Reply:
agree. done